schema/fields/date.js

```diff
@@ -2,6 +2,15 @@ import moment from 'moment';
 import * as tz from 'moment-timezone'; // eslint-disable-line no-unused-vars
 import { GraphQLString, GraphQLBoolean } from 'graphql';
 
+export function date(rawDate, format, timezone) {
+  if (timezone) {
+    if (format) return moment(rawDate).tz(timezone).format(format);
+    return moment(rawDate).tz(timezone).format();
+  }
+  if (format) return moment.utc(rawDate).format(format);
+  return rawDate;
+}
+
 export default {
   type: GraphQLString,
   args: {
```

Comment:
Can you add a description about the field and being able to set a default with the `X-TIMEZONE` header?